paddle/fluid/operators/controlflow/while_op.cc

```diff
@@ -63,9 +64,22 @@ class WhileOp : public framework::OperatorBase {
 
     framework::Executor executor(dev_place);
     auto *block = Attr<framework::BlockDesc *>(kStepBlock);
-
     auto *program = block->Program();
 
+    std::set<std::string> no_copy_var_names;
+    auto allops = block->AllOps();
+    for (auto *op : allops) {
+      auto &input_var_names = op->Inputs();
```

Comment:
Same as above, suggest to use "const VariableNameMap &" instead of "auto"

Reply:
Thanks, done.